app/controllers/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find params[:id]
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = t ".wel"
+      redirect_to @user
+    else
+      flash[:danger] = t "signup_err"
+      render "new"
+    end
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit(:name, :email, :password,
```

Comment:
tạo biến USER_PARAMS = %i(name email password password_confirmation).freeze trong model user
rồi thay nha.
```suggestion
    params.require(:user).permit User::USER_PARAMS
```